<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogsForm</class>
 <widget class="QWidget" name="CatalogsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="CatalogsSubtab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="CatalogsChairsTab">
      <attribute name="title">
       <string>Кафедры</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="4">
        <widget class="QTableWidget" name="CatalogsChairsTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Полное название кафедры</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Короткое название кафедры</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="AddChairButton">
         <property name="statusTip">
          <string>Добавить кафедру в список</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Icons/AddIcon.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="DeleteChairButton">
         <property name="statusTip">
          <string>Удалить кафедру из списка</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Icons/DeleteIcon.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsSpecialitiesTab">
      <attribute name="title">
       <string>Специальности</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="CatalogsSpecialitiesTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsClassesTab">
      <attribute name="title">
       <string>Группы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="CatalogsClassesTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsSubjectsTab">
      <attribute name="title">
       <string>Дисциплины</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableWidget" name="CatalogsSubjectsTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsRoomsTab">
      <attribute name="title">
       <string>Аудитории</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTableWidget" name="CatalogsRoomsTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsTeachersTab">
      <attribute name="title">
       <string>Преподаватели</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QTableWidget" name="CatalogsTeachersTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatalogsStudyTypesTab">
      <attribute name="title">
       <string>Виды занятий</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QTableWidget" name="StudyTypesTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
